tests/unit/test_ops.py

```diff
@@ -106,27 +106,31 @@ def test_moments(client, tmpdir, df, dataset, gpu_memory_frac, engine, op_column
 @pytest.mark.parametrize("gpu_memory_frac", [0.01, 0.1])
 @pytest.mark.parametrize("engine", ["parquet", "csv", "csv-no-header"])
 @pytest.mark.parametrize("op_columns", [["name-string"], None])
-def test_encoder(tmpdir, df, dataset, gpu_memory_frac, engine, op_columns):
+def test_encoder(tmpdir, df, dataset, gpu_memory_frac, engine, op_columns, client):
     cat_names = ["name-cat", "name-string"] if engine == "parquet" else ["name-string"]
     cont_names = ["x", "y", "id"]
     label_name = ["label"]
 
+    encoder = ops.Encoder(columns=op_columns)
     config = nvt.workflow.get_new_config()
-    config["PP"]["categorical"] = [ops.Encoder(columns=op_columns)]
+    config["PP"]["categorical"] = [encoder]
 
     processor = nvt.Workflow(
-        cat_names=cat_names, cont_names=cont_names, label_name=label_name, config=config
+        cat_names=cat_names,
+        cont_names=cont_names,
+        label_name=label_name,
+        config=config,
+        client=client,
```

Comment:
So, now that we are passing `client` here, is the single-GPU version of `Encoder` now untested?  This is certainly fine (since we are moving to the all-dask statistics backend), but perhaps we will need to remove much of the single-gpu code to pass the "coverage criteria" for CI to pass?

Reply:
We should definitely remove soon, but it's still used by the example notebooks. 

For code coverage, I've changed the 'test_gpu_workflow_api' test to test both the dask and single gpu version: https://github.com/NVIDIA/NVTabular/pull/114/files#diff-78067b78469064255f1184dc1acadb7cR37 .